magda-typescript-common/src/test/registry/buildAuthorizedClient.spec.ts

```diff
@@ -0,0 +1,44 @@
+import {} from "mocha";
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+import nock from "nock";
+import AuthorizedRegistryClient from "magda-typescript-common/src/registry/AuthorizedRegistryClient";
+
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe("Test AuthorizedRegistryClient.ts", function () {
+    afterEach(() => {
+        nock.cleanAll();
+    });
+
+    it("accepts optional userId and jwtSecret", async function () {
+        const registry = new AuthorizedRegistryClient({
+            baseUrl: "some.where",
+            userId: "some.user",
```

Comment:
As we test `accepts optional userId and jwtSecret`.  We probably should not pass `userId` to prove it's optional

Reply:
Ok, the description should be "accept userId and jwtSecret".